### ISSUE TITLE
Feature/porting request: Tab/Shift-Tab indent/unindent

### ISSUE THREAD

**Issue by singalen** [elfmz/far2l#1739]
There used to be a plugin for Far: https://github.com/char-com-ua/far-indent

No idea if it ever worked, but I really love the function in my main editor/IDE.

Let me also promote the "modern code editor" discussion: https://github.com/elfmz/far2l/discussions/1738

**Comment by elfmz** [comment]
well.. but it works under far2l - select block and press Tab - it will indent whole block, and Shift Tab unindents respectively

**Comment by singalen** [comment]
+1 oops. Thank you so much!

**Comment by singalen** [comment]
Tried it.
It's a bit hard to discover, though – the block must be a whole number of lines, a line and a half selection won't do it.
Is this intended or is it a bug?

**Comment by akruphi** [comment]
@elfmz now **Tab** & **Shift-Tab** shift block by tab (or by indent size if expand tabs is configured) **only if cursor located in start of any line**, if selection was stopped inside any line the Tab / Shift-Tab work as usual (Tab replace selected text, Shift-Tab move cursor back by tab size) :cry:

The Alt-I/Alt-U in such case working properly and shift in block all lines with partial selected.

Please correct Tab/Shit-Tab this behavior a la Alt-I/Alt-U.

**Comment by singalen** [comment]
Reopening after @akruphi's point.

**Comment by elfmz** [comment]
funny that Tab/Shift+Tab implemented in SimpleIndent plugin while Alt+I/U are native features of editor

**Comment by singalen** [comment]
I think the feature was so easy to find that it was easier to write a plugin to do it :D

**Comment by akruphi** [comment]
May be closed after #1782  or needs additional tuning?

**Comment by singalen** [comment]
Looks good to me, thank you!
